packages/accordion/src/AccordionItem.ts

```diff
@@ -23,12 +25,19 @@ import chevronIconStyles from '@spectrum-web-components/icon/src/spectrum-icon-c
 
 import styles from './accordion-item.css.js';
 
+const chevronClass = {
```

Comment:
```suggestion
const chevronIcon: Record<string, () => TemplateResult> = {```

Refer to the code for Swatch [here](https://github.com/adobe/spectrum-web-components/blob/a17290b3e1b6b0b99d68f4a77c42e9694da82982/packages/swatch/src/Swatch.ts#L34) so we can render the icon for the appropriate size up here, as opposed to in the render method.

Reply:
updated